lexicons/tools/ozone/history/defs.json

```diff
@@ -0,0 +1,166 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.history.defs",
+  "defs": {
+    "subjectBasicView": {
+      "type": "object",
+      "required": ["subject", "status", "modAction"],
+      "properties": {
+        "subject": {
+          "type": "string",
+          "format": "uri"
+        },
+        "status": {
+          "type": "string",
+          "knownValues": ["deactivated", "active", "deleted"]
+        },
+        "modAction": {
+          "type": "string",
+          "knownValues": [
+            "#modActionPending",
+            "#modActionLabel",
+            "#modActionResolve",
+            "#modActionSuspend",
+            "#modActionTakedown"
+          ]
+        },
+        "subjectProfile": {
+          "type": "ref",
+          "ref": "app.bsky.actor.defs#profileViewBasic"
+        },
+        "labels": {
+          "type": "array",
+          "items": {
+            "type": "ref",
+            "ref": "com.atproto.label.defs#label"
+          }
+        }
+      }
+    },
+    "eventView": {
+      "type": "object",
+      "required": ["subject", "createdAt", "event", "isAutomated"],
+      "properties": {
+        "subject": {
+          "type": "string",
+          "format": "uri"
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "datetime"
+        },
+        "isAutomated": {
+          "type": "boolean"
+        },
+        "event": {
+          "type": "union",
+          "refs": [
+            "tools.ozone.history.defs#eventTakedown",
+            "tools.ozone.history.defs#eventLabel",
+            "tools.ozone.history.defs#eventReport",
+            "tools.ozone.history.defs#eventEmail",
+            "tools.ozone.history.defs#eventResolve"
+          ]
+        }
+      }
+    },
+    "eventTakedown": {
+      "type": "object",
+      "description": "Permanently or temporarily taken down",
+      "properties": {
+        "comment": {
+          "type": "string"
+        },
```

Comment:
What field do the public-facing event comments get mapped from?  Mostly just want to double-check whether these are the mods' internal event comments or something else.